### ISSUE TITLE
Introduction for RGSoC Blog

### ISSUE THREAD

**Issue by katrin-k** [rails-girls-summer-of-code/Team-Hackrgirls-2015#3]
Namaste!
As you know, there are 2 blog posts you're supposed to write for the RGSoC Blog. The 1st is an introduction.

You have time until mid August. We have a [google doc](https://docs.google.com/spreadsheets/d/16QVY9lSmq-lw6YNNZTy98XkAI9Bg4S8nelM72UeWWmk/edit?usp=sharing) to organize all the teams. Please choose a date when you want to publish your blog posts. 

These are the questions for the 1st blog post:
- Why ‘Hackrgirls’? What does the name mean?
- How did you hear about the SoC and why did you apply?
- What are you working on?
- What is your happiest moment so far?
- If you could code anything in the world, what would it be?

Here is an [example from 2013](http://railsgirlssummerofcode.org/blog/meet-team-rails-grrls/). I can, of course, also help you with the blog post and check if it's ok.

Blog posts are published by doing a pull request. Here is a description how to do this https://gist.github.com/anikalindtner/153044e9bea3331cc103

In short :)
- [ ] choose a date
- [ ] write blog post
- [ ] publish blog post
